packages/openapi-generator/src/parser/unique-naming.ts

```diff
@@ -2,107 +2,70 @@ import { UniqueNameGenerator, camelCase } from '@sap-cloud-sdk/util';
 
 /**
  * Ensure uniqueness of names.
- * Takes a list of items, identifies duplicate names and renames the duplicate names.
- * @param items List of items to rename.
- * @param nameHandler Object containing a name getter and/or setter and/or formatter.
- * @param nameHandler.getName Function to get the name of an item. Retrieves the `name` property by default.
- * @param nameHandler.transformItem Function to transform the given item with the new name. Sets the `name` property by default.
- * @param nameHandler.formatName Function to transform the name when finding a unique name. Defaults to camel case.
- * @param nameHandler.reservedWords Reserved words that should be handled as duplicates.
- * @returns The given items with unique names.
+ * Takes a list of names and renames those that are duplicate.
```

Comment:
Mention that the oder is contained this is imported for later usage I think.

Reply:
Done.